README.md

```diff
@@ -1,24 +1,38 @@
 # Licensee
-_A Ruby Gem to detect under what license a project is distributed._
+
+*A Ruby Gem to detect under what license a project is distributed.*
 
 [![Build Status](https://travis-ci.org/benbalter/licensee.svg?branch=master)](https://travis-ci.org/benbalter/licensee) [![Gem Version](https://badge.fury.io/rb/licensee.svg)](http://badge.fury.io/rb/licensee)
 
 ## The problem
-- You've got an open source project. How do you know what you can and can't do with the software?
-- You've got a bunch of open source projects, how do you know what their licenses are?
-- You've got a project with a license file, but which license is it? Has it been modified?
+
+* You've got an open source project. How do you know what you can and can't do with the software?
+* You've got a bunch of open source projects, how do you know what their licenses are?
+* You've got a project with a license file, but which license is it? Has it been modified?
 
 ## The solution
+
 Licensee automates the process of reading `LICENSE` files and compares their contents to known licenses using a several strategies (which we call "Matchers"). It attempts to determine a project's license in the following order:
-- If the license file has an explicit copyright notice, and nothing more (e.g., `Copyright (c) 2015 Ben Balter`), we'll assume the author intends to retain all rights, and thus the project isn't licensed.
-- If the license is an exact match to a known license. If we strip away whitespace and copyright notice, we might get lucky, and direct string comparison in Ruby is cheap.
-- If we still can't match the license, we use a fancy math thing called the [Sørensen–Dice coefficient](https://en.wikipedia.org/wiki/S%C3%B8rensen%E2%80%93Dice_coefficient), which is really good at calculating the similarity between two strings. By calculating the percent changed from the known license to the license file, you can tell, e.g., that a given license is 90% similar to the MIT license, that 10% likely representing the copyright line being properly adapted to the project.
 
-_Special thanks to [@vmg](https://github.com/vmg) for his Git and algorithmic prowess._
+* If the license file has an explicit copyright notice, and nothing more (e.g., `Copyright (c) 2015 Ben Balter`), we'll assume the author intends to retain all rights, and thus the project isn't licensed.
+* If the license is an exact match to a known license. If we strip away whitespace and copyright notice, we might get lucky, and direct string comparison in Ruby is cheap.
```

Comment:
This paragraph is a little hard to understand, and it doesn't follow the same pattern as the previous one (if x, then y).